x-pack/plugins/ml/common/util/job_utils.ts

```diff
@@ -93,7 +93,7 @@ export function isSourceDataChartableForDetector(job: CombinedJob, detectorIndex
     // Note that the 'function' field in a record contains what the user entered e.g. 'high_count',
     // whereas the 'function_description' field holds an ML-built display hint for function e.g. 'count'.
     isSourceDataChartable =
```

Comment:
The link to the Single Metric Viewer needs disabling for jobs where the only detector is `lat_long`:

Applies to the actions in the jobs list:
![image](https://user-images.githubusercontent.com/7405507/105881989-0dee0780-5ffd-11eb-89c8-7332441614bc.png)

And the `View series` link in the anomalies table.

Reply:
Disabled link in anomalies table and removed form the charts label in 478a75725eb997b45a1c62becae5c72bb4d9a1c2

Reply:
I still see the link to the Single Metric Viewer in the jobs list, and can also select lat_long jobs in the job selection flyout in the Single Metric Viewer.